src/content/post/20161129_git_reflog_amend.md

```diff
@@ -0,0 +1,114 @@
++++
+date        = "2016-11-29T08:51:09+02:00"
+title       = "Trapped by an amend, saved by a reflog"
+tags        = ['git']
+topics      = ['git']
+description = "Personal experience on getting in trouble with a common git command and finding my way out."
+slug        = 'trapped-by-amend-saved-by-reflog'
+author      = "Marcello"
++++
+[SparkFabrik]: http://www.sparkfabrik.com/  "SparkFabrik"
+
+We use Git on a daily basis, and most of the time we always enter the same commands.
+It can happen to found ourselves trapped by one of them, sometimes because we forgot a side-effect, other times because we confused the right option.
+That's OK, because within Git most of the time lies the appropriate and easiest solution. So, let me tell you how I was hit by an `--amend` during a commit and how I easily got out.
```

Comment:
I'd rephrase this to avoid repetition (`most of the time` is already used in the opening sentence) and to make thinks clearer:

`That's OK, because within Git most of the time lies the appropriate and easiest solution.` -> `No need to worry though, since Git is equally able to put you in trouble and save your day with the same ease.`

What about that version?